processing/src/main/java/org/apache/druid/segment/incremental/ParseExceptionHandler.java

```diff
@@ -76,27 +86,51 @@ public void handle(@Nullable ParseException e)
       rowIngestionMeters.incrementProcessedWithError();
     } else {
       rowIngestionMeters.incrementUnparseable();
-    }
+      if (emitter != null) {
+        DateTime timeOfException = new DateTime(e.getTimeOfExceptionMillis());
 
-    logParseExceptionHelper(e);
-
-    if (savedParseExceptionReports != null) {
-      ParseExceptionReport parseExceptionReport = new ParseExceptionReport(
-          e.getInput(),
-          e.isFromPartiallyValidRow() ? "processedWithError" : "unparseable",
-          e.isFromPartiallyValidRow()
-          ? ((UnparseableColumnsParseException) e).getColumnExceptionMessages()
-          : ImmutableList.of(e.getMessage()),
-          e.getTimeOfExceptionMillis()
-      );
-      savedParseExceptionReports.add(parseExceptionReport);
-    }
+        ImmutableMap<String, Object> extraData = new ImmutableMap.Builder().putAll(taskMetadata)
+                                                                           .put("input", e.getInput())
+                                                                           .put("detailMessage", e.getMessage())
+                                                                           .build();
+        Event event = new AlertEvent(
+            timeOfException,
+            "peon",
+            "localhost",
+            AlertEvent.Severity.DEFAULT,
```

Comment:
The default severity appears to be `COMPONENT_FAILURE`, which isn't the case here. Consider defining a new severity type, something like `PARSING_FAILURE`